lms/djangoapps/ccx/utils.py

```diff
@@ -128,6 +128,19 @@ def get_ccx_for_coach(course, coach):
     return None
 
 
+def get_ccx_for_coach_by_ccx_id(course, coach, ccx_id):
+    """
+    Looks to see if user is coach of a CCX for this course.  Returns the CCX or
+    None.
+    """
+    ccx = CustomCourseForEdX.objects.filter(
```

Comment:
this returns a `list`. It would be better to use `.get` and handle the `CustomCourseForEdX.DoesNotExist` exception.